Get-NetView.psm1

```diff
@@ -2575,17 +2582,19 @@ function Completion {
     Maximum number of background tasks, from 0 - 16. Defaults to 5.
 
 .PARAMETER SkipAdminCheck
-    If present, the check for administrator privileges will be skipped. Note that less data
-    will be collected and the results may be of limited or no use.
+    If present, the check for administrator privileges will be skipped. Note that limited data
+    will be collected, which may be of limited use.
+    
+.PARAMETER Scenario
+    Modifies data collection to comply with special requirements for certain scenarios:
+        Universal       - Use default unmodified behavior.
+        AzureHost       - Slows down script to avoid sustained CPU spikes.
+        FailoverCluster - Skip sytem log collection.
```

Comment:
Please consider improving as follows.  Specifically to address both flexibility and simplicity.

1) !!!IMPORTANT!!!  Please ensure the Execution commands are captured.  We need to know what command was run when inspecting the *.zip file to quickly aid in diagnosis.  Capture the execution command in Sanity/Metadata.txt as a complement to the version info.

2) Separate flags should be used to control each behavior.  For example:
-ExecDelay # Command execution delay of 10ms (default)
-ExecDelay <time> # Command execution delay in ms, where range is 1ms - 1sec
-SkipLogs # Do not acquire the system diagnosis logs

2) The scenario names should be abstracted.  Simply because what is applicable to Azure today, may be applicable to AzS or HCI tomorrow.  The long term evolution/usage must be considered in proper name selection.  Additionally, using the scenario names causes unnecessary coupling of this tool to the scenarios where used.  This tool MUST remain scenario agnostic.
Consider renaming the following:
- Universal -> "Default" - Default behavior
- AzureHost -> "NonDisruptive" - internally uses -ExecDelay to achieve intent
- FailoverCluster -> IMHO, this is already clear and explicit via -SkipLogs

But honestly, I strongly believe the scenario qualification causes unnecessary complexity at this time.  Fundamentally, there's two things needed here are clear and explicit as described in (2) above.  Please consider removing the scenario logic outright.

Reply:
(1) is already covered.

Reply:
Parameters that control what data is collected are IMO antithetical to Get-NetView's purpose, which is to collect as much data as possible in one go. I am strongly opposed to adding even a single -SkipX parameter because I don't want anyone to actually use them. The scenario logic is more complex/obtuse by design to discourage use.

However, it comes with the benefit of reduced risk of complexity creep; each -SkipX parameter effectively doubles the number of data collection "scenarios", whereas Scenario logic can accomplish the same thing with the addition of a single Scenario, or by extending an existing one.

Reply:
Regarding (1).  I inspected a recent output of Metadata.txt and it does not capture the input parameters to Get-Netview.  Please show an example which has this.
- Script Version: 2020.6.15.85

Reply:
I sent you a separate email on Scenario coupling.  Let's discuss offline.